### PR TITLE
community/sacc: new aport

### PR DESCRIPTION
**Pull request by refacto**
sacc(1) is a simple console gopher client

### CODE REVIEW

File: community/sacc/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Contributor: Sascha Paunovic <azarus@posteo.net>
+# Maintainer: Sascha Paunovic <azarus@posteo.net>
+pkgname=sacc
+pkgver=1.00
+pkgrel=0
+pkgdesc="saccomys, simple console gopher client"
+url="gopher://bitreich.org/1/scm/sacc"
+arch="all"
+license="ISC"
+depends="ncurses"
```
Comment (jirutka) [inline community/sacc/APKBUILD]: Dependency on `ncurses` is autodiscovered, so it should not be specified.

```
>>> sacc*: Tracing dependencies...
	ncurses
	so:libc.musl-x86_64.so.1
	so:libncursesw.so.6   <---
```

File: community/sacc/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+source="ftp://bitreich.org/releases/$pkgname/$pkgname-v$pkgver.tgz"
+builddir="$srcdir/$pkgname-v$pkgver"
+
+build() {
+	make -C "$builddir"
+}
+
+package() {
+	cd "$builddir"
+	install -Dm644 LICENSE "$pkgdir"/usr/share/licenses/$pkgname/LICENSE
```
Comment (jirutka) [inline community/sacc/APKBUILD]: ISC is standard OSI-approved license, so it’s not needed to install the LICENSE file.

**Comment by refacto** [comment]
Thanks for the review.

**Comment by jirutka** [comment]
Also new aports must go the the testing repository first.

**Comment by jirutka** [comment]
Merged in fd1c62aa2364ad8c9cb28dec8f1edd91b7f8ec4b.

**Pull request closed without merging**